pkg/controller/controller_broker_test.go

```diff
@@ -512,7 +512,7 @@ func TestReconcileClusterServiceBrokerErrorFetchingCatalog(t *testing.T) {
 	events := getRecordedEvents(testController)
 	assertNumEvents(t, events, 1)
 
-	expectedEvent := corev1.EventTypeWarning + " " + errorFetchingCatalogReason + " " + `ClusterServiceBroker "test-broker": Error getting broker catalog: ooops`
+	expectedEvent := corev1.EventTypeWarning + " " + errorFetchingCatalogReason + " " + "Error getting broker catalog: ooops"
```

Comment:
maybe to do in a follow-up: convert this to a `fmt.Sprintf`

Reply:
I have a larger plan to refactor this type of line, working towards it.